hpython.cabal

```diff
@@ -2,167 +2,70 @@
 -- documentation, see http://haskell.org/cabal/users-guide/
 
 name:                hpython
-version:             0.0.1.0
-synopsis:            Write Python using Haskell
+version:             0.1.0.0
+-- synopsis:            
 -- description:         
-homepage:            https://github.com/qfpl/hpython
 license:             BSD3
-license-file:        LICENCE
+license-file:        LICENSE
 author:              Isaac Elliott
 maintainer:          isaace71295@gmail.com
```

Comment:
You'll get spam if this email address goes to hackage.

Recommend upside-down, or a mail alias.